deploy/charts/rawfile-csi/values.yaml

```diff
@@ -21,6 +21,11 @@ node:
   metrics:
     enabled: false
 
+storageClass:
+  enabled: false
```

Comment:
Shouldn't this be `true`? Running `k8s enable storage` will deploy this chart and the storage class would still be disabled then, right?

Reply:
Since enabling the storage class isn't something the upstream project does -- and the intent is to upstream all our changes -- then i didn't see a good reason to automatically create this storage class with the upstream charts.

I assumed `k8s enable storage` can create its override with values in the snap, right?

Reply:
@bschimke95 i hope this resolves your issue?  the `k8s` snap will override the default chart values

Reply:
It does, was more a question for understanding this better.